lib/worker/index.js

```diff
@@ -39,13 +41,15 @@ Object.keys(queues).forEach(name => {
 
     Sentry.setTag('queue', name)
     Sentry.captureException(err)
+    statsd.increment('job-error')
```

Comment:
Metric names is always a bit up in the air but for the first few in this app, I've followed this pattern: `<prefix>.<area/concern>`. This has led to metric names like:

- `jira_integration.github_requests.95th`
- `jira_integration.jira_requests.95th`

In this same vein, what do you think about creating a `jobs` namespace and adding states beneath that?

- `jira_integration.jobs.started`
- `jira_integration.jobs.completed`
- `jira_integration.jobs.errored`
- etc